### ISSUE TITLE
videojs autoplay being triggered when video is next in the gallery

### ISSUE THREAD

**Issue by mixmasterfestus** [sachinchoolur/lightGallery#616]
Hi,

When loading an item from the gallery in which the next slide is a video, that video will begin to play.

Due to what I presume is preemptive loading of the next asset it starts the video in the next slide, this became apparent when the audio played.

Happy to provide more info if necessary

Many thanks, great plugin!

Jason

**Comment by sachinchoolur** [comment]
Hey @mixmasterfestus ,
Sorry for the delayed response. It works fine for me. Here is the demo - http://sachinchoolur.github.io/lightGallery/demos/html5-videos.html#video-without-poster

Please try using the latest version of [lg-video plugin](https://github.com/sachinchoolur/lg-video)

If the issue persists please provide a jsfiddle which demonstrates the issue

**Comment by mixmasterfestus** [comment]
Hiya,

Thanks for the response, in the demo autoplay isn't enabled. When I enable it via the videoJSoptions, that is when I'm observing the behaviour

```
  container.lightGallery({
    keyPress: false,
    closable: false,
    selector: '.grid-item',
    mode: 'lg-lollipop',
    selector: '.show-in-gallery',
    videoMaxWidth: "2000px",
    videojs: true,
    videojsOptions: {
      height: 720,
      autoplay: true
    }
  });
```

Thanks again
Jason

**Comment by sachinchoolur** [comment]
Hey @mixmasterfestus ,
Sorry for the delayed response. I missed your comment.  It is related to https://github.com/sachinchoolur/lightGallery/pull/514
Will be fixed in lightGallery v2.0
